docs/management/connectors/index.asciidoc

```diff
@@ -15,4 +15,6 @@ include::action-types/tines.asciidoc[leveloffset=+1]
 include::action-types/webhook.asciidoc[]
 include::action-types/cases-webhook.asciidoc[leveloffset=+1]
 include::action-types/xmatters.asciidoc[]
+include::action-types/torq.asciidoc[]
+include::action-types/tines.asciidoc[]
```

Comment:
Hi @natexcvi the `elasticsearch-ci/docs` check is failing because the Tines docs are already included, we are including them twice, the duplicity may have been caused by some merge. 
Can you remove this line `19`? Thanks

Reply:
@semd just removed it